newIDE/app/src/MainFrame/EditorContainers/HomePage/SectionContainer.js

```diff
@@ -86,6 +95,20 @@ const SectionContainer = ({
           {children}
         </Column>
       </Paper>
+      {renderFooter && (
+        <Paper
+          elevation={0}
+          style={{
+            borderLeft: `1px solid ${GDevelopTheme.home.separator.color}`,
```

Comment:
I believe this border should have been done by the panel of the left, but not a big deal.